decidim-module-crowdfundings/spec/spec_helper.rb

```diff
@@ -4,7 +4,7 @@
 
 ENV["ENGINE_NAME"] = File.dirname(__dir__).split("/").last
 
-Decidim::Dev.dummy_app_path = File.expand_path(File.join("spec", "decidim_dummy_app"))
```

Comment:
Just to be curious, why this?

Reply:
This was initially a meaningful change since in the beginning I was sharing the dummy application for all plugins, so I needed to tweak this path to point to the repo root.

In the end I decided to use a separate dummy application for each plugin, so these changes ended up being just a normalization of how the dummy app is set (`spec/decidim_dummy_app` vs `File.join("spec", "decidim_dummy_app")`).

Reply:
yes, my doubt was why `a/b` is better than `File.join`. Performance? I understand that `File.join` improves compatibility with other OS

Reply:
Ahhh! Well, I also thought the same thing, that `File.join` is compatible accross different OS and using raw slashes is not. However, [it turns out](https://github.com/bbatsov/rubocop/issues/5823) that's not true, so raw `/` is looks way simpler to type and maintain to me.